model/gen-ai/registry.yaml

```diff
@@ -13,9 +13,9 @@ groups:
               stability: experimental
               value: "openai"
               brief: 'OpenAI'
-            - id: vertex_ai
```

Comment:
@lmolkova do you know if we can also need to mark `vertex_ai` as deprecated?  if so, how to make it? This is a value and not an attribute, did not find any example for this, thanks!